weather_dl/download_pipeline/parsers.py

```diff
@@ -18,17 +18,27 @@
 import copy as cp
 import datetime
 import json
+import logging
+import re
 import string
 import textwrap
 import typing as t
 import numpy as np
 from collections import OrderedDict
 from urllib.parse import urlparse
 
+from apache_beam.io.gcp import gcsio
+from google.api_core.exceptions import GoogleAPIError
+from google.cloud import storage
+from google.resumable_media import InvalidResponse
+
 from .clients import CLIENTS
 from .config import Config
+from .config import prepare_partitions
```

Comment:
We do allow multiple imports per line in this codebase. Feel free to make this a 1-liner.